app/src/app/core/components/top-menu/top-menu.component.spec.ts

```diff
@@ -12,6 +16,9 @@ describe('TopMenuComponent', () => {
       ],
       declarations: [
         TopMenuComponent
+      ],
+      providers: [
+        provideMockStore({ initialState }),
       ]
     })
```

Comment:
Vi saknar test för den nya knappen.

Reply:
Expanded should have correct links in TopMenuComponent with Logout and Admin test